### ISSUE TITLE
[Design Issue] Design Debt: Reorganize Figma 

### ISSUE THREAD

**Issue by akuas** [CMSgov/eAPD#3736]
Considerations
- How to display large tickets that include multiple pages in figma
- How to distinguish current implementation from MMIS design
- Integrate segmented ticket designs with overall page design
- Devs thoughts on accessing designs via figma now (connecting Storybook and Figma)
- This is not a drastic reorganization, this ticket focuses on how we organize feature tickets in figma and reactively organizing figma pages based on the plan we set out. 

We did a design debt task last year and I figure we are due for a new one. Our designs have expanded, so our current implementation designs, future enhancements, abandoned future enhancements, and segmented feature tickets all live in the same place. I think we need to figure out how to best organize our Figma pages and document it somewhere to be referenced later. I am hoping what comes out of this ticket is sort of future proofing figma designs and ensuring the context of the designs is understandable in the future.
Im also hoping once a ticket is done, feature tickets are integrated into the general flow of the page and sort of mimics our [login figma page](https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/?node-id=787%3A8).

### This task is done when…

- [x] Establish a plan to organize feature designs that span multiple tickets. Do we want all designs under one ticket banner or do we want each page of the ticket on their respective figma pages.
- [x] Establish a plan to integrate tickets into a flow on the page
- [x] Document figma organization
- [x] Organize figma pages
- [x] 508 considerations and/or recommendations are reviewed
- [ ] if there is a long discussion, a summary of changes for dev will be added to the last comment before transfer
- [ ] a list of changes to be made if this is improving an existing feature

---

**Comment by akuas** [comment]
For reference, we could note business logic and interaction design on figma, here our some resources Jenn sent me for reference. This could improve the documentation process and allow us to have design decision on hand for closed tickets.
https://www.figma.com/community/file/1092240224927774096
https://www.figma.com/community/file/981620200432052613?preview=fullscreen

**Comment by beparticular** [comment]
Progress: @akuas will create an MMIS file for future work, starting with ticket #3687 and move any MMIS specific pages or work from the current Figma file to a new Figma file.

**Comment by akuas** [comment]
@beparticular Unfortunately, I didn't get to finish wrapping this up but will do as soon I come back from my break. It's a little hard to understand now, but you are welcome to add to it, if you want. 
https://docs.google.com/document/d/1YIk44CY0Woy9IjblSevMFytnYo_60eHR9ksocfGWoK8/edit

**Comment by akuas** [comment]
@jeromeleecms The figma reorganization plan is ready to go. @beparticular Can you please take a look and let me know if there is anything Im missing in this write up.
https://docs.google.com/document/d/1YIk44CY0Woy9IjblSevMFytnYo_60eHR9ksocfGWoK8/edit#heading=h.4buoe1gxg1xx

**Comment by akuas** [comment]
I moved this file to the eAPD documents: Design and Research folder. Below is the updated link
https://docs.google.com/document/d/1oneKYrIArJENzx-l0ItUD19yvy9DmXIsYJ-pXi1IuKs/edit?usp=sharing

**Comment by beparticular** [comment]
This is complete as far as setting up how we want to move forward with Figma. 
Other Figma work is represented in other tickets.
